tuist/tuist
# Migrate info plist generator to a project mapper

Continues #1206 

### Short description 📝

We now have model mappers that are capable of updating Projects and Targets as well as include side effects such are adding or removing files. 

_@pepibumur already started this work in #1206 - this is a continuation of that work._

### Solution 📦

Migrate info plist generator to a model mapper.

### Implementation 👩‍💻👨‍💻

- [x] Update #1206 to leverage model mappers 

### Test Plan 🛠

- Run `tuist generate` within `fixtures/ios_app_with_watchapp2`
- Verify the Info.plist files continue to be generated to the `Derived/InfoPlists` directory
